cloud-deployment-guide/cloud-troubleshooting.md

```diff
@@ -0,0 +1,103 @@
+
+Sometimes the domain in the **Referer** header does not match with the URL in the browser's address bar. To check the **Referer** header:
+1. Open your browser's _Developer's Tools_.
+1. Open the _Network_ tab.
+1. Find and select the request being made to load {{site.productname}} from {{site.cloudname}} with your API key.
+1. Click on the **Headers** tab.
+
+In the section called **Request Headers** there should be a field for **Referer**. This is the value that {{site.productname}} is checking against your approved domains. It must match one of your approved domains listed on your [{{site.accountpage}}]({{site.accountpageurl}}).
+
+### Solution
+
+If the `Referer` is correct for the site, ensure that the domain is included in your list of approved domains on [{{site.accountpage}}]({{site.accountpageurl}}). If the `Referer` is not what you are expecting, you may need to adjust your application's [**Referer** header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) settings.
+
+## "We’re unable to check your domain because the referer header is missing. Please read the Guide on how to ensure your referer header is present, so we can then customize your editor experience."
+
+### Cause
+
+This notification is shown if the [**Referer** header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) is absent for the network request for loading {{site.productname}} from {{site.cloudname}}. {{site.cloudname}} verifies the domain {{site.productname}} is loading from by checking the domain of the **Referer** header in the network request.
+
+*Referer* headers are sometimes removed by browser settings or browser extensions. {{site.cloudname}} only needs the domain in the **Referer** header, so for improved performance and privacy {{site.companyname}} recommends setting the `referrerpolicy` to `origin` when requesting {{site.cloudname}} resources.
+
+### Solution
+
+Identify which browser setting or extension is responsible for blocking the **Referer** being sent. A common extension is `Referer Control`:
+
+- [Google Chrome](https://chrome.google.com/webstore/detail/referer-control/hnkcfpcejkafcihlgbojoidoihckciin?hl=en)
+- [Firefox](https://addons.mozilla.org/en-US/firefox/addon/referercontrol/)
+
+Once you have identified the setting or extension, modify it to allow just the `Origin` of the `Referer` to be sent. Alternatively, disable the extension or setting for any pages using {{site.cloudname}}.
+
+## "The ___ premium plugin is not enabled on your API key. Upgrade your account."
+
+### Cause
+
+This notification is shown when your apiKey does not have access to the premium plugin being requested. This could be the result of a trial expiring, and your {{site.productname}} configuration attempting to load premium plugins you can no longer access.
```

Comment:
```suggestion
This notification is shown when your API key does not have access to the premium plugin being requested. This could be the result of a trial expiring, and your {{site.productname}} configuration attempting to load premium plugins you can no longer access.
```
I'm assuming at this point this is the right version to be using.